Principal_component_analysis/test/Principal_component_analysis/test_linear_least_squares_fitting_circles_2.cpp

```diff
@@ -1,3 +1,5 @@
+#define CGAL_NO_DEPRECATION_WARNINGS
```

Comment:
This will create errors on the no-deprecated-code test suite, please see https://github.com/CGAL/cgal/pull/3096.

Reply:
Thanks, I fixed it.